### ISSUE TITLE
ROI距离计算

### ISSUE THREAD

**Issue by maxxB-130** [Cherishzyh/ProstrateECE#2]
您好，我在复现您关于前列腺癌包膜外侵犯的文章时，在计算ROI距离的热力图处遇到了一些困难，您调用的BasicTool目录下的文件代码不知可否借阅一下（from BasicTool.MeDIT.Visualization import ShowColorByRoi；      from BasicTool.MeDIT.ArrayProcess import ExtractPatch, ExtractBlock；        from ECEDataProcess.DataProcess.MaxRoi import GetRoiCenter, GetRoiCenterBefore）不胜感谢

**Comment by Cherishzyh** [comment]
你好，如果只是要生成Distance Map的话使用DistanceMap/RoiDistanceMap.py中的FindRegion函数就可以了，可以见DistanceMap/RoiDistanceMap.py中的Test，不需要使用到上述的函数哦~上述的是我自己用来查看生成图像的。谢谢~

**Comment by maxxB-130** [comment]
好的，感谢

**Comment by maxxB-130** [comment]
您好，我想请问一下您最终参与训练/测试的带CBAM的ResNeXt网络模型和PAGNet网络模型是哪个目录下的什么文件，MyModel还是SYECE？最近几次实验得出的结果感觉指标提不上去，我怀疑也许是我网络内部有些问题没处理（可能看的网络模型只是初稿？），还请告知，谢谢！

**Comment by Cherishzyh** [comment]
您好，我最终用的训练代码都在SYECE里，PAGNet模型是SYECE/model.py/ResNeXt, 带CBAM的模型是SYECE/ModelWithoutDis.py/ResNeXt。

**Comment by maxxB-130** [comment]
明白了，感谢

**Comment by maxxB-130** [comment]
请教您一个问题，实验中的多层面图像是不是不需要单独进行训练，直接在inference阶段利用leading slice训练好的模型输出预测结果？我之前用多层面图像作为样本训练了模型QAQ

**Comment by Cherishzyh** [comment]
> 

是的。我们在处理的时候并没有用多层的数据去重新训练，而是直接inference。比较了一下最大癌灶层和多层的inference结果是并没有明显差异的~这也说明了其实模型对部分噪声数据是鲁棒的。其实条件允许的话你可以试试3D的模型~

**Comment by maxxB-130** [comment]
好的，谢谢
